Detections/ThreatIntelligenceIndicator/IPEntity_AzureSQL.yaml

```diff
@@ -0,0 +1,52 @@
+id: d0aa8969-1bbe-4da3-9e76-09e5f67c9d85
+name: TI map IP entity to Azure SQL Security Audit Events
+description: |
+  'Identifies a match in SQLSecurityAuditEvents from any IP IOC from TI'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: ThreatIntelligence
+    dataTypes:
+      - ThreatIntelligenceIndicator
+  - connectorId: ThreatIntelligenceTaxii
+    dataTypes:
+      - ThreatIntelligenceIndicator
+queryFrequency: 1h
+queryPeriod: 14d
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - Impact
```

Comment:
Is there no technique we can map to?

Reply:
We don't do technique's for IOC matching, we decided early on with Jason Wescott that we would leave it off.  Happy to revisit again, but okay for now.